tests/modules/deepvariant/test.yml

```diff
@@ -0,0 +1,9 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml deepvariant
+- name: deepvariant
+  command: nextflow run ./tests/modules/deepvariant -entry test_deepvariant -c tests/config/nextflow.config
+  tags:
+    - deepvariant
+  files:
+    - path: output/deepvariant/test.bam
+      md5sum: e667c7caad0bc4b7ac383fd023c654fc
```

Comment:
I think you should generate a new test with the changes so this md5sum updates along with it :P

Reply:
Hehe, definitely :)